src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1438,6 +1438,16 @@ From there, simply click on a Markdown file, PDF file or SVG icon in the File Ex
   <data name="Oobe_Welcome" xml:space="preserve">
     <value>Welcome</value>
   </data>
+  <data name="Oobe_WhatsNew" xml:space="preserve">
+    <value>What's new?</value>
+  </data>
+  <data name="Oobe_WhatsNew_LoadingError.Title" xml:space="preserve">
+    <value>Couldn't load the release notes. Please check your internet connection.</value>
```

Comment:
@niels9001 
I think we should move "internet connection" sentence into message description to have a better layout.